src/oidcop/oidc/token.py

```diff
@@ -109,20 +35,25 @@ def process_request(self, req: Union[Message, dict], **kwargs):
         _log_debug = logger.debug
 
         if req["grant_type"] != "authorization_code":
-            return self.error_cls(
-                error="invalid_request", error_description="Unknown grant_type"
-            )
+            return self.error_cls(error="invalid_request", error_description="Unknown grant_type")
 
         try:
             _access_code = req["code"].replace(" ", "+")
         except KeyError:  # Missing code parameter - absolutely fatal
-            return self.error_cls(
-                error="invalid_request", error_description="Missing code"
-            )
+            return self.error_cls(error="invalid_request", error_description="Missing code")
 
         _session_info = _mngr.get_session_info_by_token(_access_code, grant=True)
         grant = _session_info["grant"]
 
+        token_type = "Bearer"
+
+        # Is DPOP supported
+        if "dpop_signing_alg_values_supported" in _context.provider_info:
```

Comment:
I think that it would be clearer if we had a flag for enabling DPoP instead. E.g. `dpop_enabled`.

Of course that flag shouldn't be exposed in the provider_info.

Reply:
OK, we can put in endpoint_context.

Reply:
Pushed a modification with this.